packages/playground/src/components/pull-request/utils.tsx

```diff
@@ -167,3 +167,26 @@ export enum ViewStyle {
   SIDE_BY_SIDE = 'side-by-side',
   LINE_BY_LINE = 'line-by-line'
 }
+
+//**
+//  * parses diff strings to extract the starting line number from the hunk header. this
+//  * function is needed because the diff viewer being used hides that info because it assumes
+//  * the starting line is not relevant for the displayed diffs, leading to potential confusion for users.
+//  * @param diffString: string
+//  * @returns string or null
+//
+export function parseStartingLineIfOne(diffString: string) {
+  // Use a regular expression to find the hunk header in the diff string
+  const hunkHeaderMatch = diffString.match(/@@ -(\d+),\d+ \+\d+,\d+ @@/)
+  if (hunkHeaderMatch) {
+    const startingLine = parseInt(hunkHeaderMatch[1], 10) // Convert the extracted string to an integer
```

Comment:
Do we know what is `10` here?

Reply:
oh i dont need it, it was just to make sure it converts to base 10 hahah @vardanbansal-harness